src/spikeinterface/postprocessing/template_similarity.py

```diff
@@ -13,25 +13,22 @@ class ComputeTemplateSimilarity(AnalyzerExtension):
 
     Similarity is defined as 1 - distance(T_1, T_2) for two templates T_1, T_2
 
-
     Parameters
     ----------
     sorting_analyzer : SortingAnalyzer
         The SortingAnalyzer object
     method : str, default: "cosine"
         The method to compute the similarity. Can be in ["cosine", "l2", "l1"]
+        In case of "l1" or "l2", the formula used is:
```

Comment:
I can't comment this but above could we give the options rather than have the type `str`

Reply:
Done!